Source/Core/DolphinQt/Config/Mapping/MappingIndicator.h

```diff
@@ -69,4 +69,6 @@ class CalibrationWidget : public QToolButton
   QAction* m_completion_action;
   ControllerEmu::ReshapableInput::CalibrationData m_calibration_data;
   QTimer* m_informative_timer;
+
+  bool is_centering = false;
```

Comment:
Private members should be prefixed with `m_` per our code-style.

Reply:
Fixed in latest revision